### DISCUSSION TITLE
Zerotier repository for CasaOS?

### DISCUSSION THREAD

**Discussion by Andersonong-github** [hassio-addons/app-zerotier#226] [General]
Hi Sir,

1st of all thank you very much for this repository for home assistant & I like it very much. just would like to suggest that if you could make it available for CasaOS as well? this wonderful addon for CasaOS.

**Comment by frenck** [comment]
Sorry, I don't use or work with CasaOS.

This repository provides a Home Assistant add-on.

../Frenck
